### ISSUE TITLE
[Accessibility]: Disabled form controls are read as "dimmed"

### ISSUE THREAD

**Issue by efritz** [sourcegraph/sourcegraph#35190]
### Audit type

Screen reader navigation

### User journey audit issue

#33516

### Problem description

<img width="884" alt="dimmed" src="https://user-images.githubusercontent.com/103087/167504685-5b2fad93-3e82-473b-ac53-34aad25dfe97.png">

When navigating over disabled form controls the are read as "dimmed".

### Expected behavior

I didn't expect to not be able to edit this. It doesn't explicitly say disabled. Is this a screen reader convention that's already understood and I'm just new to the experience?

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
